charts/workflow-controller/templates/rbac.yaml

```diff
@@ -3,24 +3,40 @@ kind: ClusterRole
 metadata:
   name: workflow-controller
 rules:
-- apiGroups:
-  - apiextensions.k8s.io
+- apiGroups: ["apiextensions.k8s.io"]
   resources:
   - customresourcedefinitions
-  verbs:
-  - "*"
-- apiGroups:
-  - {{ .Values.api.groupName }}
+  verbs: ["*"]
+- apiGroups: [{{ .Values.apiGroupName | quote }}]
   resources:
   - workflows
-  verbs:
-  - "*"
-- apiGroups:
-  - batch
+  verbs: ["*"]
+- apiGroups: ["batch"]
   resources:
   - jobs
   verbs: ["*"]
 - apiGroups: [""]
   resources:
   - namespaces
-  verbs: ["list"]
\ No newline at end of file
+  verbs: ["list"]
+
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: workflow-controller
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: workflow-controller
+subjects:
+- kind: ServiceAccount
+  name: {{ .Values.serviceAccount }}
+  namespace: kube-system
+
+---
+apiVersion: v1
```

Comment:
why did you remove the service-account.yaml file and add everything in this file?

My understanding with `helm` is that it is more readable to have on file by a resource.

Reply:
No special reason. Can split back into 3 files: CR, CRB and SA. Just thought it would be more readable to see all of them on the same page (files are very small).
Do you want me to split them back?